cts-java/src/test/java/org/ga4gh/cts/api/Utils.java

```diff
@@ -117,7 +117,10 @@ public static String randomName() {
      * @throws AvroRemoteException is the server throws an exception or there's an I/O error
      */
     public static String getValidReferenceId(Client client) throws AvroRemoteException {
-        final SearchReferenceSetsRequest refSetsReq = SearchReferenceSetsRequest.newBuilder().build();
+        final SearchReferenceSetsRequest refSetsReq =
+                SearchReferenceSetsRequest.newBuilder()
+                                          .setAssemblyId(TestData.REFERENCESET_ASSEMBLY_ID)
+                                          .build();
         final SearchReferenceSetsResponse refSetsResp = client.references.searchReferenceSets(refSetsReq);
 
         final List<ReferenceSet> refSets = refSetsResp.getReferenceSets();
```

Comment:
Actually I would just replace this whole refset section with a call to the other helper:
`getReferencedSetIdByAssemblyId`